bin/downgrade

```diff
@@ -113,12 +115,20 @@ read_selection() {
 prompt_to_ignore() {
   local pkg ans
 
+  if [[ $DOWNGRADE_PROMPT_IGNORE == never ]]; then
+    return
+  fi
+
   for pkg; do
     if ! pacignore check -c "$PACMAN_CONF" "$pkg"; then
-      eval_gettext "add \$pkg to IgnorePkg? [y/N] "
-      read -r ans
-      if [[ "${ans,,}" == $(gettext 'y')* ]]; then
+      if [[ $DOWNGRADE_PROMPT_IGNORE == always ]]; then
         pacignore add -c "$PACMAN_CONF" "$pkg"
+      else
+        eval_gettext "add \$pkg to IgnorePkg? [y/N] "
+        read -r ans
+        if [[ "${ans,,}" == $(gettext 'y')* ]]; then
+          pacignore add -c "$PACMAN_CONF" "$pkg"
+        fi
       fi
     fi
   done
```

Comment:
Ah this was the reason why `test/prompt_to_ignore/user_approves.t` is the way it is:

https://github.com/archlinux-downgrade/downgrade/pull/197#issuecomment-1128621802

Reply:
Well, I ended up with calling `pacignore`.... I don't like unit tests and mocking anyway because integration tests need less code and less updates, while debugging shell scripts isn't as hard as C.

Reply:
> while debugging shell scripts isn't as hard as C.

Oops, unless stderr is discarded and a vague message is given instead... However, I think the test failure on CI is due to there is no pacman-conf command. I can fix this if you accept doing the tests this way.

Reply:
We already have integration tests for `pacignore` in `tests/pacignore`, so I think it makes sense to maintain the tests in `tests/prompt_to_ignore` as unit tests by mocking `pacignore`. Tests pass now.